tests/framework/ROM/SKLearn/gold/data/outOVO.csv

```diff
@@ -1,122 +1,122 @@
 X,Y,Z,ProbabilityWeight
-2.0,-1000.0,1,0.0025
-2.0,-800.0,1,0.005
-2.0,-600.0,1,0.005
-2.0,-400.0,1,0.005
-2.0,-200.0,1,0.005
-2.0,0.0,1,0.005
-2.0,200.0,1,0.005
-2.0,400.0,1,0.005
-2.0,600.0,1,0.005
+2.0,-1000.0,0,0.0025
```

Comment:
Could you provide a comment why all values of z are zeros instead of ones?

Reply:
I think because the OVO classifier is using:
```
    <ROM name="estimator" subType="LinearRegression">
      <Features>X,Y</Features>
      <Target>Z</Target>
      <fit_intercept>True</fit_intercept>
      <normalize>True</normalize>
    </ROM>
``` 
underneath, and a linear regression is not a good fit for modeling this.
As I mentioned in the commit, this is because of this change: https://github.com/scikit-learn/scikit-learn/pull/22604

Reply:
I changed the estimator to Gaussian naive Bayes and the values are now a mixture of zeros and ones.